notebooks/official/tabular_workflows/bqml_arima_plus.ipynb

```diff
@@ -730,18 +698,18 @@
         "\n",
         "The training pipeline expects the following parameters:\n",
         "\n",
-        "- `bigquery_destination_uri`: (optional) BigQuery Dataset URI. Used to export the metrics table and model. If not given, we will create one for the user.\n",
+        "- `bigquery_destination_uri`: (optional) BigQuery Dataset URI. Used to export the metrics table and model. If not given, we can create one for the user.\n",
         "- `data_granularity_unit`: Enum used to specify the time granularity (hour, day, week, month, etc).\n",
         "- `data_source_csv_filenames` or `data_source_bigquery_table_path`: A URI for either a CSV stored in GCR or a BigQuery table, respectively.\n",
-        "- `evaluated_examples_destination_uri\t`: (optional) BigQuery Dataset URI OR Table URI. Used to export the evaluated examples table. Will use bigquery_destination_uri if not provided.\n",
+        "- `evaluated_examples_destination_uri`: (optional) BigQuery Dataset URI OR Table URI. Used to export the evaluated examples table. Uses bigquery_destination_uri if not provided.\n",
```

Comment:
add comma after "bigquery_destination_uri"
- "...bigquery_destination_uri, if not provided"